runbotics/runbotics-scheduler/src/scheduler-database/process/process-crud.service.ts

```diff
@@ -188,7 +223,7 @@ export class ProcessCrudService {
                 tenantId: user.tenantId,
                 id,
             },
-            relations: RELATIONS,
+            relations: relations,
```

Comment:
relations: RELATIONS (if you decide that this variable name should be capitalized) or you can shorted this to just relations (no need for relations: relations)